### DISCUSSION TITLE
Sort products by quantity in stock and sold quantities

### DISCUSSION THREAD

**Discussion by romanxpo3** [Smile-SA/elasticsuite#2571] [Q&A]
Hello,

Is it possible to add relevancy rules (with optimizers ?) based on the quantity in stock (not only in stock/out of stock, but the real quantity number), and sold quantities (number of sales on a given time) ?

I did not find how to do it.

Thank you.
Roman

**Comment by tuyennn** [comment]
In my opinion, this is possible but there are more indexing should be created.
